### PR TITLE
Step reorder

### PR DESCRIPTION
**Pull request by SavageWilliam**
Fixes issues in the review from other pull request
& issue #47

### CODE REVIEW

**Comment by des-des** [comment]
@SavageWilliam the response to the review goes on the pull with the review? Not sure I get this..

**Comment by des-des** [comment]
Am happy to merge but will cause conflicts on other pull.

File: create-ladder.js [hunk 1]
```diff
@@ -1,65 +1,89 @@
...
+      TweenLite.set(node, { x: 0 });
+      boxes[i] = {
+        transform: node._gsTransform,
+        x: node.offsetLeft,
+        y: node.offsetTop,
+        node
+      };
+    }
+    group.classList.toggle("reorder");
+    console.log('group classlist: ', group.classList);
```
Comment (des-des) [inline create-ladder.js]: :fire:

File: create-ladder.js [hunk 2]
```diff
@@ -1,65 +1,89 @@
...
+    console.log(group.classList);
```
Comment (des-des) [inline create-ladder.js]: :fire:

File: create-ladder.js [hunk 3]
```diff
@@ -1,65 +1,89 @@
...
+  const layout = () => {
+    console.log('complete clicked');
+    var group = document.querySelector(".group");
+    var nodes = document.querySelectorAll(".box");
+    var total = nodes.length;
+    var ease  = Power1.easeInOut;
+    var boxes = [];
 
-window.onload = showLadder, addHandlersToElements;
+    for (var i = 0; i < total; i++) {
```
Comment (des-des) [inline create-ladder.js]: why node nodes.forEach((node, i) =>

File: create-ladder.js [hunk 4]
```diff
@@ -1,65 +1,89 @@
...
-  const cloudM = document.getElementById('cloudM');
-  cloudM.style.display ='none'
-  //title
-  const cloudTitle = document.getElementById('cloudTitle');
-  cloudTitle.className += ' animated zoomOut'
-  ladder.className += ' animated fadeOut'
-})
+  //REORDER steps
+  const layout = () => {
+    console.log('complete clicked');
```
Comment (des-des) [inline create-ladder.js]: :fire:

**Comment by SavageWilliam** [comment]
If you're happy with changes we cna get this onto master, as it cover the other pull requests changes

**Comment by des-des** [comment]
@SavageWilliam it is much harder for me to see if you hav responded to my comments if you resolve them on a different pull..

**Pull request merged**